{
  "@context": "https://openvex.dev/ns",
  "@id": "https://openvex.dev/docs/sysdig/vex-427ad2254b739b01dd993c09e805f7d0655638010e8522da97a20b2a96d9aed1",
  "author": "atomist/sysdig-integration@0.1.3",
  "role": "Document Creator",
  "version": "1",
  "statements": [
    {
      "vulnerability": "CVE-2019-12900",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/bzip2@1.0.6-r6?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2022-45143",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:maven/org.apache.tomcat.embed/tomcat-embed-core@9.0.55"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2021-43980",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:maven/org.apache.tomcat.embed/tomcat-embed-core@9.0.55",
        "pkg:maven/org.apache.tomcat.embed/tomcat-embed-websocket@9.0.55"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2201",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/libjpeg-turbo@1.5.3-r3?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-14498",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/libjpeg-turbo@1.5.3-r3?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-5094",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/e2fsprogs@1.44.2-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-5188",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/e2fsprogs@1.44.2-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-1000654",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/libtasn1@4.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-20217",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/krb5@1.15.3-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14578",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2803",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14593",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2978",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2981",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2805",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2698",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2945",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2992",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2949",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14556",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2659",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2602",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2757",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2987",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2842",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-11212",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-16435",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2988",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2989",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14803",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-3214",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2781",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2958",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14798",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14781",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14779",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2583",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-3139",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-3183",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-13785",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-3169",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14792",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2755",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2983",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14583",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2769",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-3149",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2964",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2745",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-3180",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2973",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2422",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2754",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14797",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2766",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2601",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2593",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2654",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-7317",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8",
        "pkg:apk/alpine/libpng@1.6.34-r1?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2894",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2684",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2426",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2800",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2962",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2975",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2830",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14577",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2604",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2816",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2756",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14796",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14782",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14581",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2590",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2762",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-2773",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2999",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-3136",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2786",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14579",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-2933",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-14621",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/openjdk8@8.181.13-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-20346",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/sqlite@3.24.0-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-8457",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/sqlite@3.24.0-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2020-11655",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/sqlite@3.24.0-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-19242",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/sqlite@3.24.0-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-19244",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/sqlite@3.24.0-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2019-16168",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/sqlite@3.24.0-r0?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-14550",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/libpng@1.6.34-r1?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    },
    {
      "vulnerability": "CVE-2018-14048",
      "products": [
        "pkg:docker/atomist/vulnapp@sha256:9c8a98c1f53866b3d8c121481424a78a9e9473334fd64ce29e7823dac9ea9c64"
      ],
      "subcomponents": [
        "pkg:apk/alpine/libpng@1.6.34-r1?os_name=alpine&os_version=3.8"
      ],
      "status": "not_affected",
      "justification": "vulnerable_code_not_in_execute_path",
      "impact_statement": "Vulnerable package not loaded at runtime"
    }
  ]
}
